tests/unit/neptune/new/client/test_run.py

```diff
@@ -117,7 +116,7 @@ def test_entrypoint(self):
             self.assertEqual(exp["source_code/entrypoint"].fetch(), "main_dir/main.py")
 
         with init_run(mode="debug", source_files=["internal/*"]) as exp:
-            self.assertEqual(exp["source_code/entrypoint"].fetch(), "main.py")
+            self.assertEqual(exp["source_code/entrypoint"].fetch(), "../main.py")
```

Comment:
Why do we have to change it?

Reply:
I had to remove the `os.getcwd` patch, as it was messing up path resolving in operation storage. Without it, the entrypoint in this case evaluates to `../main.py`. I investigated it and this patch was a little strange, as it also changed the behaviour of `get_common_root` and produced a rather unexpected output. 

The `../main.py` output makes more sense to me, cause the 'common_root' of the source_code var is `cwd + internal/`. The entrypoint path is `cwd + main.py`. So the relative path of entrypoint, where the start is common_root is `../main.py`. Check out `internal/utils/source_code.py` file, `upload_source_code` func